packages/e2e-tests/specs/editor/various/toolbar-roving-tabindex.test.js

```diff
@@ -13,11 +13,17 @@ async function focusBlockToolbar() {
 }
 
 async function expectLabelToHaveFocus( label ) {
-	await expect(
-		await page.evaluate( () =>
+	let ariaLabel = await page.evaluate( () =>
+		document.activeElement.getAttribute( 'aria-label' )
+	);
+	// If the labels don't match, try pressing Up Arrow to focus the block wrapper in non-content editable block.
```

Comment:
I think this is way simpler compared to the modifications that were here and tests are still green. Seems logical to me. This whole file still is in desperate need of commenting.